src/popup/locales/en.json

```diff
@@ -1156,6 +1156,7 @@
     "nameUnregistered": "This name has already been registered.",
     "notSameAs": "Sender and receiver address are the same. You will send tokens to your own account.",
     "tokenToAnAddress": "Unfortunately you can only send AEX-9 Tokens to an address",
+    "tokenToAnBtcAddress": "Invalid BTC address",
```

Comment:
I suggest to not use any protocol specific translations. Maybe you can use dynamic properties?

Reply:
BTC validation check only address, in AE is checking also the .chainame. I think this should be separated.